.travis.yml

```diff
@@ -1,14 +1,21 @@
 language: python
-python: 2.7
+
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+  - "3.8"
+  - "3.8-dev"
 
 before_install:
   - sudo apt update -y
   - sudo apt-get -y install lua5.1 luarocks
   - sudo luarocks install luacheck
 
 install:
-  - pip install -r requirements.txt
-  - pip install -r requirements-test.txt
+  - pip3 install -r requirements.txt
+  - pip3 install -r requirements-test.txt
```

Comment:
Is `pip3` actually necessary? Whether that means that just `pip` does not correspond to the Python version from the matrix above? We use just `python` in the `Makefile`, not `python3`, and I guess it works. So using of `pip3` looks lopsided.

I unable to verify, Travis CI is gone. So, if there is a reason to the change, I'm okay.

Reply:
> Is pip3 actually necessary? 

`pip3` used here to explicitly run it with python3. Some environment may have pip that points to pip2 actually.

Reply:
If you are still confused and not satisfied by comment above I can check it for at least env on test-run CI and replace pip3 to pip if everything well there. But again, our CI is broken :(

Reply:
I expect that Travis CI setup correct environement, when `python: [<versions>]` is used. However, yep, there is no simple way to check it, so feel free to skip this comment.

BTW, since you added 2.7, there is another reason to use `pip` here.

Reply:
> BTW, since you added 2.7, there is another reason to use pip here.

Yep, replaced `pip3` to `pip`.